common/workspace-hack/Cargo.toml

```diff
@@ -63,6 +63,7 @@ tokio-util = { version = "0.6.7", features = ["codec", "compat", "default", "fut
 toml = { version = "0.5.8", features = ["default"] }
 tracing = { version = "0.1.26", features = ["attributes", "default", "log", "std", "tracing-attributes"] }
 tracing-core = { version = "0.1.18", features = ["default", "lazy_static", "std"] }
+ureq = { version = "1.5.4", default-features = false, features = ["charset", "encoding", "json", "native-tls", "serde", "serde_json"] }
```

Comment:
Now that we are using `reqwest`, plz remove `ureq` from the Cargo.toml/.lock files.